app/routes/items/search_order.js

```diff
@@ -84,5 +84,12 @@ export default AuthorizeRoute.extend({
     }
     controller.set("searchText", "");
     controller.set("backLinkPath", this.get("itemDesignateBackLinkPath"));
+  },
+
+  resetController(controller, isExiting) {
+    if (isExiting) {
+      controller.set("partial_qty", undefined);
```

Comment:
why are we setting undefined?? I would very much prefer null

Reply:
On doing `null` url does not remove these params and remains with value `Nan`. It removed it on assigning `undefined`.

Reply:
Ember object should really have an `unset` method. There's no clear way of deleting a key :/

Reply:
Just noticed this `controller.set("isSet", undefined);` 

--> could we make the variable name of `isSet` a bit more explicit ? I'm not really sure that is set or not 😆

Reply:
right @patrixr, the `isSet` property is not very self-explanatory. @abulsayyad123 could you please make this something understandable ?

Reply:
@patrixr , just got to know
set is collection of packages.. eg =>  Dining table is set which is made of 4 chairs and 1 table
you will find set_item_id column on package table maybe. 
isSet checks that the package is a set or not. 

cc - @namrataukirde @abulsayyad123